pkg/datapath/tables/route.go

```diff
@@ -53,12 +54,12 @@ type RouteID struct {
 }
 
 func (id RouteID) Key() index.Key {
-	key := append(index.Uint64(uint64(id.Table)), '+')
-	key = append(key, index.Uint64(uint64(id.Table))...)
-	key = append(key, '+')
-	key = append(key, index.NetIPPrefix(id.Dst)...)
-	key = append(key, 0 /* termination */)
-	return key
+	key := make([]byte, 0, 4 /* table */ +4 /* link */ +17 /* prefix & bits */)
```

Comment:
Instead of these inline comments why not define each int as a constant? Could be way easier to read.